src/fileio.cpp

```diff
@@ -392,15 +393,17 @@ static void TarAddLink(const std::string &srcParam, const std::string &destParam
  * Replace '/' by #PATHSEPCHAR, and force 'name' to lowercase.
  * @param name Filename to process.
  */
-static void SimplifyFileName(char *name)
+static void SimplifyFileName(std::string &name)
 {
-	/* Force lowercase */
-	strtolower(name);
-
-	/* Tar-files always have '/' path-separator, but we want our PATHSEPCHAR */
+	std::transform(name.begin(), name.end(), name.begin(), [](char c) {
```

Comment:
seems mildly overkill, why not a for loop? iirc you did that for SanitizeFilename elsewhere

Reply:
It's what you get when looking for a function to get lower case characters. But I changed it back to a for loop.